core/cmd/app.go

```diff
@@ -620,12 +620,30 @@ func NewApp(client *Client) *cli.App {
 							Action: client.VersionDatabase,
 							Flags:  []cli.Flag{},
 						},
+						{
+							Name:   "status",
+							Usage:  "Display the current database migration status.",
+							Action: client.StatusDatabase,
+							Flags:  []cli.Flag{},
+						},
 						{
 							Name:   "migrate",
 							Usage:  "Migrate the database to the latest version.",
 							Action: client.MigrateDatabase,
 							Flags:  []cli.Flag{},
 						},
+						{
```

Comment:
Amazing, any chance of a rollback command?

Reply:
Done! `chainlink node db rollback` will roll back a single migration, `chainlink node db rollback <version>` will roll back to a specific version.